README.md

```diff
@@ -8,8 +8,8 @@
 2. Ensure that curl is installed on the system by running `curl --version`
    - If curl is not installed, run `sudo apt install curl` to install it
 ### Running the Script
-1. Run the command `curl -sSL https://github.com/VOLTTRON/volttron-installer/blob/main/web.py | python3.10 -`
-   - Installs the ansible and git packages if they are not already installed
+1. Run the command `python3 <(curl -sSL https://raw.githubusercontent.com/VOLTTRON/volttron-installer/develop/web.py)`
```

Comment:
So now that this goes to main then the develop is wrong...perhaps use <BRANCH> for the curl command?